packages/cryptoplease/lib/features/backup_phrase/module.dart

```diff
@@ -45,9 +45,24 @@ class _ContentState extends State<_Content> {
   @override
   void initState() {
     super.initState();
-    context
-        .read<PuzzleReminderBloc>()
-        .add(const PuzzleReminderEvent.checkRequested());
+
+    final accessMode = context.read<AccountsBloc>().state.accessMode;
```

Comment:
Let's rather put this logic into `PuzzleReminderBloc` – pass `AccessMode` as a parameter in `PuzzleReminderEvent.checkRequested()`.